### ISSUE TITLE
Gourmet doesn't find plugin

### ISSUE THREAD

**Issue by marillat** [kirienko/gourmet#264]
Debian unstable python 3.8 amd64

Plugin are not loaded and not seen by gourmet

```
Plugin  unit_converter not found
Plugin  duplicate_finder not found
Plugin  spellcheck not found
Plugin  gxml_plugin not found
Plugin  html_plugin not found
Plugin  mastercook_import_plugin not found
Plugin  mealmaster_plugin not found
Plugin  archive_plugin not found
Plugin  pdf_plugin not found
Plugin  plaintext_plugin not found
Plugin  web_import_plugin not found
Plugin  website_import_plugins not found
Plugin  krecipe_plugin not found
Plugin  mycookbook_plugin not found
Plugin  epub_plugin not found
```
strace display 

```
openat(AT_FDCWD, "/usr/lib/python3/dist-packages/gourmet/plugins/import_export", O_RDONLY|O_NONBLOCK|O_CLOEXEC|O_DIRECTORY) = 6
```
But plugins are there 
```
$ ls -l /usr/lib/python3/dist-packages/gourmet/plugins/import_export
total 52
drwxr-xr-x 3 root root 4096 20 oct.  10:51 archive_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 epub_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 gxml_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 html_plugin
-rw-r--r-- 1 root root    0 15 oct.  00:08 __init__.py
drwxr-xr-x 3 root root 4096 20 oct.  10:51 krecipe_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 mastercook_import_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 mealmaster_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 mycookbook_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 pdf_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 plaintext_plugin
drwxr-xr-x 2 root root 4096 20 oct.  10:51 __pycache__
drwxr-xr-x 3 root root 4096 20 oct.  10:51 web_import_plugin
drwxr-xr-x 3 root root 4096 20 oct.  10:51 website_import_plugins
```

**Comment by cydanil** [comment]
Thanks, could you tell me if you find any `*.gourmet-plugin` files?
```sh
find . -type f \( -name "*.gourmet-plugin" \)
```

You should have such an output:
```
./gourmet/plugins/unit_converter.gourmet-plugin
./gourmet/plugins/python_shell.gourmet-plugin
./gourmet/plugins/listsaver.gourmet-plugin
./gourmet/plugins/import_export/html.gourmet-plugin
./gourmet/plugins/import_export/gxml.gourmet-plugin
./gourmet/plugins/import_export/mealmaster.gourmet-plugin
./gourmet/plugins/import_export/epub.gourmet-plugin
./gourmet/plugins/import_export/pdf.gourmet-plugin
./gourmet/plugins/import_export/mycookbook_plugin.gourmet-plugin
./gourmet/plugins/import_export/mastercook_plugin.gourmet-plugin
./gourmet/plugins/import_export/webimport.gourmet-plugin
./gourmet/plugins/import_export/plaintext.gourmet-plugin
./gourmet/plugins/import_export/krecipe_plugin.gourmet-plugin
./gourmet/plugins/import_export/website_import.gourmet-plugin
./gourmet/plugins/import_export/archive.gourmet-plugin
./gourmet/plugins/field_editor.gourmet-plugin
./gourmet/plugins/spellcheck.gourmet-plugin
./gourmet/plugins/unit_display_prefs.gourmet-plugin
./gourmet/plugins/duplicate_finder.gourmet-plugin
./gourmet/plugins/browse_plugin.gourmet-plugin
./gourmet/plugins/key_editor.gourmet-plugin
./gourmet/plugins/nutritional_information.gourmet-plugin
./gourmet/plugins/utf16.gourmet-plugin
./gourmet/plugins/email.gourmet-plugin
./gourmet/plugins/shopping_associations.gourmet-plugin
```

If not, you probably missed building the internationalization first: `python setup.py build_i18n`.  
I gather that you installed Gourmet from having cloned this repository?

**Comment by zombiewoof** [comment]
Sorry to jump in on this issue, but I, too, seem to be missing plugins. I ran the find command provided, above, but nothing was found.

I used flatpak to install.

Can you tell me how to get the plugins installed?

**Comment by marillat** [comment]
> If not, you probably missed building the internationalization first: `python setup.py build_i18n`.

I'm using the release tarball preview2 and running `python setup.py build_i18n` solve this issue.

Why build_i18n isn't called when the package is built ?

**Comment by cydanil** [comment]
This archive is created by Github itself upon tagging, not by us.  
I understand that your use case is to build a debian package (really cool!), I'll see what can be done (probably we have to create it ourselves).

**Comment by cydanil** [comment]
Our CI builds a source version with internationalization, although it's meant to be used by `pip` directly.  
I've added it to the release page here: https://github.com/kirienko/gourmet/releases/tag/v1-alpha2  (`gourmet-2db9db8f.tar.gz`)

Does that suit you, or do you need a internationalized version of the repo itself?

**Comment by marillat** [comment]
It's fine for me now I know that build_i18n need to be called.

Debian packages are available in experimental (0.17.5~alpha2-3)
I think I'll upload a new package to unstable next week. 

https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=gourmet

Thanks
Christian
